ksqldb-engine/src/main/java/io/confluent/ksql/services/SandboxedSchemaRegistryClient.java

```diff
@@ -36,21 +46,158 @@
 
   static SchemaRegistryClient createProxy(final SchemaRegistryClient delegate) {
     Objects.requireNonNull(delegate, "delegate");
+    final SandboxSchemaRegistryCache sandboxSchemaRegistryCache = new SandboxSchemaRegistryCache(delegate);
 
     return LimitedProxyBuilder.forClass(SchemaRegistryClient.class)
-        .swallow("register", anyParams(), 123)
+        .forward("register", methodParams(String.class, ParsedSchema.class), sandboxSchemaRegistryCache)
```

Comment:
This is the "main" fix -- instead of swallowing the registration, we capture it within the sandbox.

Reply:
how are we planning to test this?

Reply:
how are we planning to test this?

Reply:
Working on tests... The PR is still WIP. Just opened if for early feedback.